.github/workflows/deploy.yml

```diff
@@ -173,9 +180,6 @@ jobs:
       - register-runner
     if: ${{ always() && !cancelled() && needs.deploy.result == 'success' && github.ref_name != 'production' }}
     runs-on: ubuntu-latest
-    container:
-      image: cypress/browsers:node18.12.0-chrome107
-      options: --user 1001
```

Comment:
🤔

Reply:
This breaks stuff.  It's not a real user, so it has no real permissions; no only does it not scope the paths to the workflow, but it also prevents executing container functions.